packages/perseus-core/src/widgets/table/table-util.ts

```diff
@@ -0,0 +1,13 @@
+import type {PerseusTableWidgetOptions} from "@khanacademy/perseus-core";
+
+type TableWidgetPublicOptions = Pick<
```

Comment:
I believe we've been naming them `<widget>PublicWidgetOptions` so this would be `TablePublicWidgetOptions`

Reply:
Good catch, thanks!